kwok/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.access.redhat.com/ubi8/go-toolset:1.19 AS builder
+
+WORKDIR /src
+
+COPY go.mod go.sum ./
+
+RUN go mod download
+
+COPY . .
+
+RUN go install sigs.k8s.io/kwok/cmd/kwok@v0.3.0
+RUN go install sigs.k8s.io/kwok/cmd/kwokctl@v0.2.1
+
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+COPY --from=builder /opt/app-root/src/go/bin/kwok /usr/local/bin/
+COPY --from=builder /opt/app-root/src/go/bin/kwokctl /usr/local/bin/
+
+RUN command -v kwok && command -v kwokctl
```

Comment:
I guess this is for testing? Probably shouldn't stay in the final version...

Also what about setting up a cluster inside the image?

Reply:
Done